SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -4,12 +4,12 @@
 
 #include "Interop/Connection/SpatialWorkerConnection.h"
 
-FSpatialOutputDevice::FSpatialOutputDevice(USpatialWorkerConnection* InConnection, FString LoggerName)
+FSpatialOutputDevice::FSpatialOutputDevice(USpatialWorkerConnection* InConnection, FString LoggerName, int32 InPIEIndex)
+	: FilterLevel(ELogVerbosity::Warning)
+	, Connection(InConnection)
+	, Name(LoggerName)
+	, PIEIndex(InPIEIndex)
```

Comment:
Instead of storing PIEIndex, maybe just set bLogToSpatial = false if PIEIndex > -1?